src/api/java/com/minecolonies/api/util/constant/WindowConstants.java

```diff
@@ -1090,6 +1090,41 @@
     public static final String INPUT_QTY    = "qty";
     public static final String BUTTON_REQUEST = "request";
 
+    /**
+     * Id of the done button in the GUI.
+     */
+    public static final String BUTTON_DONE = "done";
+
+    /**
+     * Id of the citizen name in the GUI.
+     */
+    public static final String CITIZEN_LABEL = "citizen";
+
+    /**
+     * Id of the citizen list in the GUI.
+     */
+    public static final String CITIZEN_LIST = "unassigned";
+
+    /**
+     * Assign button of the gui.
+     */
+    public static final String CITIZEN_DONE = "done";
```

Comment:
Why is the string "done" twice there

Reply:
because it's two different buttons, so the id might change for one but not for both at th esame time

Reply:
Ok, I was looking how many dupes are in window constants - there are fewer than I expected :D